### ISSUE TITLE
Assess whether warning icons should use alert role

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1442]
assess whether or not the alert icons (such as insufficient project points) should use the alert role causing their rendering to interrupt any other text being read by a screenreader.

**Comment by rmmayo** [comment]
the warning icons already use the alert role, although the chrome plugin screen reader only seems to read them when the page is initially loaded
